test/e2e/hazelcast_wan_slow_test.go

```diff
@@ -0,0 +1,402 @@
+package e2e
+
+import (
+	"context"
+	"math"
+	"strconv"
+	. "time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"k8s.io/apimachinery/pkg/types"
+
+	hazelcastcomv1alpha1 "github.com/hazelcast/hazelcast-platform-operator/api/v1alpha1"
+	hazelcastconfig "github.com/hazelcast/hazelcast-platform-operator/test/e2e/config/hazelcast"
+)
+
+var _ = Describe("Hazelcast WAN", Label("hz_wan_slow"), func() {
+	waitForLBAddress := func(name types.NamespacedName) string {
+		By("waiting for load balancer address")
+		hz := &hazelcastcomv1alpha1.Hazelcast{}
+		Eventually(func() string {
+			hz := &hazelcastcomv1alpha1.Hazelcast{}
+			err := k8sClient.Get(context.Background(), name, hz)
+			Expect(err).ToNot(HaveOccurred())
+			return hz.Status.ExternalAddresses
+		}, 3*Minute, interval).Should(Not(BeEmpty()))
+		Expect(k8sClient.Get(context.Background(), name, hz)).To(Succeed())
+		return hz.Status.ExternalAddresses
+	}
+
+	BeforeEach(func() {
+		if !useExistingCluster() {
+			Skip("End to end tests require k8s cluster. Set USE_EXISTING_CLUSTER=true")
+		}
+		if runningLocally() {
+			return
+		}
+	})
+
+	AfterEach(func() {
+		for _, ns := range []string{hzSourceLookupKey.Namespace, hzTargetLookupKey.Namespace} {
+			DeleteAllOf(&hazelcastcomv1alpha1.WanReplication{}, &hazelcastcomv1alpha1.WanReplicationList{}, ns, labels)
+			DeleteAllOf(&hazelcastcomv1alpha1.Map{}, &hazelcastcomv1alpha1.MapList{}, ns, labels)
+			DeleteAllOf(&hazelcastcomv1alpha1.Hazelcast{}, &hazelcastcomv1alpha1.HazelcastList{}, ns, labels)
```

Comment:
I think we are using exposeExternally here, maybe we should not wait for deletion of Hazelcast resource because it will take some time.
```suggestion
			DeleteAllOf(&hazelcastcomv1alpha1.Hazelcast{}, nil, ns, labels)
```

Reply:
Fixed and it works